File: GamesCatalog/GamesCatalog/Game.swift
//
//  Game.swift
//  GamesCatalog
//
//  Created by Ilya Kovalenko on 2/12/18.
//  Copyright © 2018 Илья Коваленко. All rights reserved.
//

import Foundation
import UIKit

class Game
{
    init(title : String, releaseDate : String, shortDescription : String, fullDescription : String, url : String, image : UIImage?, fullImage : UIImage?)
    {
        Title = title
        ReleaseDate = releaseDate
        ShortDescription = shortDescription
        FullDescription = fullDescription
        Url = url
        Image = image
        FullImage = fullImage
    }
    
    public var Title : String
    public var ReleaseDate : String
    public var ShortDescription : String
    public var FullDescription : String
    public var Url : String
    public var Image : UIImage?
    public var FullImage : UIImage?
}

File: GamesCatalogAndroid/app/src/main/java/me/webasm/gamescatalogandroid/Game.java
package me.webasm.gamescatalogandroid;

import java.io.Serializable;

/**
 * Created by ikovalenko on 2/26/18.
 */

class Game implements Serializable {
    private String title;
    private String releaseDate;
    private String shortDescription;
    private String fullDescription;
    private String url;
    private String image;
    private String fullImage;

    public String getTitle() {
        return title;
    }

    public String getReleaseDate() {
        return releaseDate;
    }

    public String getShortDescription() {
        return shortDescription;
    }

    public String getFullDescription() {
        return fullDescription;
    }

    public String getUrl() {
        return url;
    }

    public String getImage() {
        return image;
    }

    public String getFullImage() {
        return fullImage;
    }

    Game(String title, String releaseDate, String shortDescription, String fullDescription,
         String url, String image, String fullImage) {
        this.title = title;
        this.releaseDate = releaseDate;
        this.shortDescription = shortDescription;
        this.fullDescription = fullDescription;
        this.url = url;
        this.image = image;
        this.fullImage = fullImage;
    }
}
